### DISCUSSION TITLE
Update failed with exit code -2067529717

### DISCUSSION THREAD

**Discussion by RadhikaGaddam** [dataplat/dbatools#9087] [General]
Hi,

I am getting below error when I apply SQL patch SQLServer2019-KB5027702-x64. Command failing with the exit code -2067529717. any Ideas?

WARNING: [16:04:10][Invoke-DbaAdvancedUpdate] Update failed with exit code -2067529717
WARNING: [16:04:11][Update-DbaInstance] Update failed: Update failed with exit code -2067529717

**Comment by potatoqualitee** [comment]
Thank you for the report. Please update dbatools and try with the latest version. See this issue if you get an authenticode error: https://github.com/dataplat/dbatools/issues/9090 (basically, just do what it says and add the -SkipPublisherCheck param)

**Comment by RadhikaGaddam** [comment]
Thanks Chrissy.
I will update and try.
Thanks,Radhika Gaddam
    On Thursday, 21 September, 2023 at 10:02:13 pm GMT+10, Chrissy LeMaire ***@***.***> wrote:  
 
 


Thank you for the report. Please update dbatools and try with the latest version. See this issue if you get an authenticode error: #9090 (basically, just do what it says and add the -SkipPublisherCheck param)
